site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,294 @@
+# Policy Attachment
+
+Policies attached to Gateway API resources and implementations should use the
+following approach to ensure consistency across implementations of the API.
+There are three primary components of this pattern:
+
+* A shared means of attaching policy to resources.
+* Support for configuring both default and override values within policy
+  resources.
+* A shared hierarchy to illustrate how default and override values should
```

Comment:
```suggestion
* A hierarchy to illustrate how default and override values should
```

Not sure if shared really helps here. NBD.

Reply:
Updated, thanks!